src/lambda/commands/deleteLambda.ts

```diff
@@ -44,13 +45,16 @@ export async function deleteLambda({
     if (!deleteParams.functionName) {
         return
     }
+    const startTime = new Date()
+    let deleteResult: result = 'Succeeded'
```

Comment:
Is this and `'Failed'` a part of a type? If not, should it be?

Reply:
Yes it's part of the `result` type that deleteResult is declared to be